frontend/tests/utils/helpers.ts

```diff
@@ -1,14 +1,30 @@
-import { TEST_URL } from "tests/common/constants";
+import { TEST_EMAIL, TEST_URL } from "../common/constants";
 import { getText } from "./selectors";
 
 export async function goToPage(url: string = TEST_URL) {
   await page.goto(url);
 }
 
-export async function login(url: string = TEST_URL) {
-  await goToPage(url);
-  await page.click(getText("Log In"));
-  await page.waitForNavigation({ url, waitUntil: "networkidle" });
+export async function login() {
+  try {
+    await expect(page).toHaveSelector(getText("My Collections"));
+  } catch (error) {
+    const password = "Test1111";
```

Comment:
is this account created in auth0?

Reply:
Yep!

Reply:
this value shouldn't be in the public repo, especially if this user also exists on prod.

Reply:
https://github.com/chanzuckerberg/corpora-data-portal/issues/900

Reply:
hmm what's the downside of exposing this prod password?

Reply:
[Here's](https://www.beyondtrust.com/blog/entry/hardcoded-and-embedded-credentials-are-an-it-security-hazard-heres-what-you-need-to-know) a good article on why exposing hardcoded passwords is a bad idea with some case studies.